src/Features/Analytics.php

```diff
@@ -112,114 +112,151 @@ public function register_cache_clear_tool( $debug_tools ) {
 	 * Registers report pages.
 	 */
 	public function register_pages() {
-		$navigation_enabled = Loader::is_feature_enabled( 'navigation' );
+		$report_pages = self::get_report_pages();
+		foreach ( $report_pages as $report_page ) {
+			if ( ! is_null( $report_page ) ) {
+				wc_admin_register_page( $report_page );
+			}
+		}
+	}
 
+	/**
+	 * Get report pages.
+	 */
+	public static function get_report_pages() {
 		$overview_page = array(
-			'id'           => 'woocommerce-analytics',
-			'title'        => __( 'Analytics', 'woocommerce-admin' ),
-			'path'         => '/analytics/overview',
-			'icon'         => 'dashicons-chart-bar',
-			'position'     => 56, // After WooCommerce & Product menu items.
-			'order'        => 10,
-			'is_category'  => true,
-			'is_top_level' => true,
+			'id'       => 'woocommerce-analytics',
+			'title'    => __( 'Analytics', 'woocommerce-admin' ),
+			'path'     => '/analytics/overview',
+			'icon'     => 'dashicons-chart-bar',
+			'position' => 56, // After WooCommerce & Product menu items.
 		);
 
 		$report_pages = array(
 			$overview_page,
 			array(
-				'id'     => 'woocommerce-analytics-overview',
-				'title'  => __( 'Overview', 'woocommerce-admin' ),
-				'parent' => 'woocommerce-analytics',
-				'path'   => '/analytics/overview',
-				'order'  => 10,
+				'id'       => 'woocommerce-analytics-overview',
+				'title'    => __( 'Overview', 'woocommerce-admin' ),
+				'parent'   => 'woocommerce-analytics',
+				'path'     => '/analytics/overview',
+				'nav_args' => array(
+					'order'  => 10,
+					'parent' => 'analytics',
```

Comment:
@joshuatf, this is a nice addition with `nav_args`. Small change, but I'm thinking `id` could also go under `nav_args` since its used only for matching a slot with the client's fill.